README.md

```diff
@@ -13,4 +10,12 @@ cd js_school_diary
 npm install
 
 npm run dev
-```
\ No newline at end of file
+```
+
+## Technology Stack:
+[![StackShare](https://img.shields.io/badge/tech-stack-0690fa.svg?style=flat)](https://stackshare.io/stackshare/awesome-stacks)
+
+- #### Development environment: <br>**Vite** (https://vitejs.dev/)
+- #### Language:<br><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Typescript_logo_2020.svg/160px-Typescript_logo_2020.svg.png" width="24"> **TypeScript** (https://www.typescriptlang.org/) 
```

Comment:
husky. eslint, https://www.conventionalcommits.org/en/v1.0.0/, 
write about all instruments plz

Reply:
from readme I'd like to get how use conventionalcommits for example

Reply:
Done